### ISSUE TITLE
Get Zalenium work with AWS ALB Ingress

### ISSUE THREAD

**Issue by ramapalani** [zalando/zalenium#958]
## 🐛 Bug Report

Get Zalenium work with AWS ALB Ingress
.Values.ingress.path https://github.com/zalando/zalenium/blob/master/charts/zalenium/values.yaml#L160 is set to "/" as default.  With this setting, aws-alb ingress is not able to route request to the selenium server running on 4445.  

One option to get this working is to set the path to "/*", but .Values.ingress.path is used in other places where * is causing problem (eg. livenessProbe: https://github.com/zalando/zalenium/blob/master/charts/zalenium/templates/_pod-template.yaml#L50; readinessProbe:https://github.com/zalando/zalenium/blob/master/charts/zalenium/templates/_pod-template.yaml#L62)

## To Reproduce

Steps to reproduce the behavior (including the docker command/docker-compose/Kubernetes manifests to start Zalenium):

Set the values.yaml to bring aws-alb ingress
```
ingress:
  enabled: true
  tls: false
  # secretName: my-tls-cert # only needed if tls above is true
  hostname: ipteam1.rp.iks.a.intuit.com
  # If ALB set host name through external-dns.alpha.kubernetes.io/hostname
  # hostname: 
  # If your ingress host name is shared with multiple
  # applications, Enter a path starting with / (eg. /zalenium)
  # If your ingress host name is only for Zalenium,
  # set the path as / (root)
  #path: /
  annotations:
    # kubernetes.io/ingress.class: "nginx"
    # kubernetes.io/tls-acme: "true"
    kubernetes.io/ingress.class: aws-alb
    alb.ingress.kubernetes.io/subnets: IngressSubnetAz1, IngressSubnetAz2, IngressSubnetAz3
    alb.ingress.kubernetes.io/target-type: instance
    alb.ingress.kubernetes.io/scheme: internet-facing
    alb.ingress.kubernetes.io/backend-protocol: HTTP
    alb.ingress.kubernetes.io/security-groups: iks-intuit-oip-cidr-ingress-tcp-80
    external-dns.alpha.kubernetes.io/hostname: ipteam1.rp.iks.a.intuit.com
```

## Expected behavior

http[s]://hostname/grid/console should work

## Test script reproducing this issue (when applicable)

Please provide a test script to reproduce the issue you are reporting, if the 
setup is more complex, GitHub repo links are also OK.

Issues without a reproduction script are likely to stall and eventually be closed.

## Environment

OS: 
Zalenium Image Version(s):  tag 3
Docker version: 18.09.2
Docker-Compose version (if applicable):
Exact Docker command to start the containers (if using docker-compose, provide 
the docker-compose file as well):
If using Kubernetes, specify your environment, and if relevant your manifests:
Kubernetes on AWS

**Comment by ramapalani** [comment]
Here is the PR for this issue https://github.com/zalando/zalenium/pull/965

**Comment by diemol** [comment]
Closed via #965, thanks @ramapalani!
